app/models/case/sar/offender.rb

```diff
@@ -56,6 +56,7 @@ def searchable_fields_and_ranks
     detainee: 'detainee',
     ex_detainee: 'ex_detainee',
     probation_service_user: 'probation_service_user',
+    ex_probation_service_user: "ex_probation_service_user"
```

Comment:
Really small thing but I generally in ruby double quotes are only needed / used with string interpolation. I think it makes sense for these to be single quotes like the others as it is easier to read as well 👍